### DISCUSSION TITLE
Example with Brush and bar plots

### DISCUSSION THREAD

**Discussion by HansKristianVikOlsen** [airbnb/visx#1596] [General]
Is it possible to use the Brush functionality with bars instead of lines/Area with timescale and working zoom label change from day/month/year? 

Exactly like this example but with bars: https://airbnb.io/visx/brush

**Comment by williaster** [comment]
Hi, I'm converting this to a discussion since it's not an issue.

**Comment by tturnerswdev33** [comment]
Any examples?
